### PR TITLE
optimize auto-detected paths

### PR DESCRIPTION
**Pull request by achrefbensaad**
optimize auto-detected paths

### CODE REVIEW

File: src/common/pathAggregator.go [hunk 1]
```diff
@@ -167,12 +167,17 @@ func buildPathTree(treeMap map[string]*Node, paths []string) {
 
 		rootPath := tokenizedPaths[0]
 
+		if rootPath == "/tmp" {
```
Comment (seswarrajan) [inline src/common/pathAggregator.go:170]: Why are we specifically adding tmp for check here @achrefbensaad ?

Comment (achrefbensaad) [reply src/common/pathAggregator.go:170]: tmp files creates by applications under /tmp have random names and are not consistant between pod restarts, so we need to allow /tmp/**.
Adding tmp there is the simplest way to guarenty that behavior.

**Review by seswarrajan — APPROVED** [review]
LGTM

**Comment by Ankurk99** [comment]
> tmp files creates by applications under /tmp have random names and are not consistant between pod restarts, so we need to allow /tmp/**.
Adding tmp there is the simplest way to guarenty that behaviour

But we have also seen this behaviour with the "var" folder too (both /var/ and /var) enteries exist, right?

**Comment by achrefbensaad** [comment]
> But we have also seen this behaviour with the "var" folder too (both /var/ and /var) enteries exist, right?

Thats in the summary not in the wpfs

**Review by Ankurk99 — APPROVED** [review]
You may want to take a look at the go-lint warning. Otherwise LGTM :+1:

**Review by nyrahul — APPROVED** [review]

**Pull request merged**